src/main/java/net/snowflake/client/core/SFBaseResultSet.java

```diff
@@ -283,7 +283,7 @@ protected SfSqlArray getJsonArray(String obj, int columnIndex) throws SFExceptio
       int columnType = ColumnTypeHelper.getColumnType(columnSubType, session);
       int scale = fieldMetadata.getScale();
 
-      ArrayNode arrayNode = (ArrayNode) OBJECT_MAPPER.readTree(obj);
+      ArrayNode arrayNode = (ArrayNode) OBJECT_MAPPER.readTree(obj.replaceAll("undefined", "null"));
```

Comment:
I don't think we should make such changes...

Reply:
I are right. Removed